### DISCUSSION TITLE
External Item Entity type (Microsoft Search Data Source for ServiceNow)  doesn't show result

### DISCUSSION THREAD

**Discussion by Robyskariah** [microsoft-search/pnp-modern-search#1928] [Q&A]
I have an External Data Source in Microsoft Search Data Sources. Connects ServiceNow Knowledge Base. When using in PnP Modern Search webpart do not display any result. Though total count returns accurately. Am I missing anything here ?

![image](https://user-images.githubusercontent.com/23713114/157810481-ef7a1f23-f050-4f19-8af7-acea13054261.png)

**Comment by wobba** [comment]
You need to adjust the slots and selected properties you are using to ensure the display templates can show information. 

https://microsoft-search.github.io/pnp-modern-search/usage/search-results/slots/
